tools/torch_quant/torch_quant/mnn/MNN_compression_pb2.py

```diff
@@ -0,0 +1,701 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Add comment to clarify that we only put disc header here for linter, and briefly explain how to generate and update this schema using proto file from MNN repo.

Reply:
Or add a README.md under torch_quant/mnn folder for this?

Reply:
Done.